qr-code-auth/README.md

```diff
@@ -0,0 +1,277 @@
+# Collection - QrCodeAuth
+
```

Comment:
are we going to create a seperate route for these api's or these endpoints will come under the `auth` route?

Reply:
if yes then the route will be `/auth/qr-code-auth`, correct?